content/en/xtc/xtc-release-notes/v72.md

```diff
@@ -0,0 +1,56 @@
+---
+title: XTC v72
+linkTitle: XTC v72
+
+weight: 144
+type: docs
+
+date: 2023-09-18
+
+description: 2023-09-19
+
+sitemap:
+  changefreq: never
+  priority: 0.1
+---
+
+## Platform
+
+### New Feature: Project Documentation
+
+Projects now provide a documentation page. See the new entry *Documentation* in the navigation bar. Use this page to share general information about the project with other project members, for example, testing goals and success criteria, test windows, contact information, news, etc. The content of the page can be formatted nicely using Markdown.
+
+Members with the role of *Reviewer* (and higher) can view the documentation page. In order to edit the page, you need to be *Test Manager* or *Project Administrator*.
```

Comment:
```suggestion
Members having the project role *Reviewer* (and higher) can view the documentation page. In order to edit the page, you need to have project role *Test Manager* or *Project Administrator*.
```

Reply:
Online grammar/style checkers replaced "having" with "with", so I suggest to trust them with this.